awesome_owl/static/src/playground.xml

```diff
@@ -2,9 +2,19 @@
 <templates xml:space="preserve">
 
     <t t-name="awesome_owl.playground">
-        <div class="p-3">
-            hello world
-        </div>
+        <Counter onChange.bind="incrementSum"/>
+        <Counter onChange.bind="incrementSum"/>
+        <Counter onChange.bind="incrementSum"/>
+        <Counter onChange.bind="incrementSum"/>
+        <Counter onChange.bind="incrementSum"/>
```

Comment:
It can probably be wrapped in a `<t t-foreach="[1, 2, 3, 4, 5]" t-as="item" t-key="item">`